### PR TITLE
update to latest fastify and all plugin to latest release;  configure…

### PR DESCRIPTION
**Pull request by smartiniOnGitHub**
… tap tests to run in parallel; fix lint errors (from the updated version of the linter).

Note that I didn't change the release of the plugin, it need ot be done after.
Last the v0.2.0 tag is missing.

Bye,
Sandro

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -18,14 +18,15 @@ test('Should return 503 on maxEventLoopDelay', t => {
...
+  fastify.listen(0, (err, address) => {
     t.error(err)
+    fastify.server.unref()
 
     // Increased to prevent Travis to fail
     process.nextTick(() => sleep(1000))
     sget({
       method: 'GET',
-      url: 'http://localhost:' + fastify.server.address().port
+      url: `${address}`
```
Comment (delvedor) [inline test.js]: You can use just `address`, there is no need to use a template string :)

Comment (smartiniOnGitHub) [reply test.js]: @delvedor removed template strings where not really needed ... thanks for the feedback

**Comment by smartiniOnGitHub** [comment]
@cemremengu Hi, should I do even the change in the README, or do you prefer to do it (other than the new plugin release number) ?

**Comment by cemremengu** [comment]
@smartiniOnGitHub you can do it in this PR I think. It will be just a single line in readme

**Comment by smartiniOnGitHub** [comment]
@cemremengu updated README, hope it's good enough. Bye

**Comment by cemremengu** [comment]
@smartiniOnGitHub merge+publish done :)

**Pull request merged**